sys/cddl/contrib/opensolaris/uts/common/sys/dtrace.h

```diff
@@ -981,6 +1011,17 @@ typedef struct dtrace_eprobedesc {
 	dtrace_recdesc_t dtepd_rec[1];		/* records themselves */
 } dtrace_eprobedesc_t;
 
+#ifdef COMPAT_FREEBSD64
+typedef struct dtrace_eprobedesc_64 {
```

Comment:
```suggestion
typedef struct dtrace_eprobedesc64 {
```

and a bunch more like this.